pdc/apps/repository/models.py

```diff
@@ -108,18 +108,6 @@ def clean(self):
             raise ValidationError("Found '-fast' in repo name '%s', but release type is '%s'"
                                   % (self.name, release_type))
 
-        if release_type == "eus" and ".z" not in self.name:
-            raise ValidationError("Missing '.z' in repo name '%s'" % self.name)
-        if release_type != "eus" and ".z" in self.name:
-            raise ValidationError("Found '.z' in repo name '%s', but release type is '%s'"
-                                  % (self.name, release_type))
-
-        if release_type == "aus" and (".aus" not in self.name and ".ll" not in self.name):
-            raise ValidationError("Missing '.aus' or '.ll' in repo name '%s'" % self.name)
-        if release_type != "aus" and (".aus" in self.name or ".ll" in self.name):
-            raise ValidationError("Found '.aus' or '.ll' in repo name '%s', but release type is '%s'"
-                                  % (self.name, release_type))
-
```

Comment:
I believe we said these checks would be based on configuration option - not deleted but being able to turn them on/off